app/static/js/sensors.js

```diff
@@ -0,0 +1,56 @@
+var app = angular.module('solarsenseApp', []);
+
+app.config(['$interpolateProvider', function($interpolateProvider) {
+  	$interpolateProvider.startSymbol('{a');
+  	$interpolateProvider.endSymbol('a}');
+}]);
+
+app.controller('LinkCtrl', function($scope, $window) {
```

Comment:
Same thing here, make sure to switch this to use the global navigation bar.

Reply:
This has been resolved with merging the change in dev into this branch